core/optaplanner-core-impl/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -394,25 +393,4 @@ public void visitReferencedClasses(Consumer<Class<?>> classVisitor) {
         }
     }
 
-    private boolean isUsingDrools() {
-        if (scoreDrlList != null || scoreDrlFileList != null) { // We know we're in DRL.
-            return true;
-        } else if (constraintProviderClass == null) { // We know we're neither in DRL nor in CS.
-            return false;
-        }
-        return (constraintStreamImplType == null || constraintStreamImplType == ConstraintStreamImplType.DROOLS);
-    }
-
-    public boolean isDroolsAlphaNetworkCompilationEnabled() {
```

Comment:
Can we just remove a public method from the config class (backward compatibility)?

Reply:
I have no idea how many users (if any) check this option via the `SolverConfig` API. Deprecation is a safe bet, though.

We check the config via Revapi, but in this particular case the excludes might not be configured properly:
https://github.com/kiegroup/optaplanner/blob/main/core/optaplanner-core-impl/src/build/revapi-config.json#L7